lib/grid.rb

```diff
@@ -0,0 +1,39 @@
+require_relative './grid_creator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.